src/main/java/org/broadinstitute/hellbender/utils/read/GATKReadToBDGAlignmentRecordConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.function.Function;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import org.bdgenomics.adam.converters.SAMRecordConverter;
+import org.bdgenomics.adam.models.SequenceDictionary;
+import org.bdgenomics.adam.models.RecordGroupDictionary;
+
+
+import java.io.Serializable;
+
+/**
+ * Converts a GATKRead to a BDG AlignmentRecord
+ */
+public class GATKReadToBDGAlignmentRecordConverter implements Function<GATKRead, AlignmentRecord> {
```

Comment:
Ok, changed in the incoming commit.  Note that I created two static methods, as the `convert` method needs the header and two other objects that are computed from the header.  But it would be wasteful to recompute the objects on each call to convert.